packages/@aws-cdk/aws-servicecatalog/lib/tag-options.ts

```diff
@@ -1,14 +1,56 @@
+import * as cdk from '@aws-cdk/core';
+import { hashValues } from './private/util';
+import { InputValidator } from './private/validation';
+import { CfnTagOption } from './servicecatalog.generated';
+
+// keep this import separate from other imports to reduce chance for merge conflicts with v2-main
+// eslint-disable-next-line no-duplicate-imports, import/order
+import { Construct } from 'constructs';
+
 /**
- * Defines a Tag Option, which are similar to tags
- * but have multiple values per key.
+ * Properties for TagOptions.
  */
-export class TagOptions {
+export interface TagOptionsProps {
   /**
-  * List of CfnTagOption
-  */
-  public readonly tagOptionsMap: { [key: string]: string[] };
+   * TagOption keys and associated list of possible values.
```

Comment:
It can be confusing with keys in a map struct and keys to a tag, but I think we should stick with using tag key as the wording.  I don't know if I've seen tags referenced with a name and not just key-value.